php/commands/media.php

```diff
@@ -18,6 +18,9 @@ class Media_Command extends WP_CLI_Command {
 	 * [--skip-delete]
 	 * : Skip deletion of the original thumbnails. If your thumbnails are linked from sources outside your control, it's likely best to leave them around. Defaults to false.
 	 *
+	 * [--only-missing]
+	 * : Regenerate only media files that have some generated size missing. No effect if used without --skip-delete.
```

Comment:
Yeah, could do, I was more thinking about adding a standalone functionality not affecting any other switches or so. But yes, it's possible and it would make sense.